### DISCUSSION TITLE
Can't connect to MySQL server on '127.0.0.1' (115)

### DISCUSSION THREAD

**Discussion by webmagnets** [postalserver/postal#3170] [Help with using Postal]
When I try `postal initialize` I get an error that says "Can't connect to MySQL server on '127.0.0.1' (115)". 

(I don't know what MySQL has to do with anything. I thought it would use MariaDB.) Anyway, can some kind soul please help me get past this error?

**Comment by willpower232** [comment]
The libraries still refer to MySQL as the protocol is the same (more or less).

Have you set up Maria and filled out the config file?

**Reply by webmagnets** [reply]
Yes, I did it according to the instructions. Of course, I may have messed it up.

**Reply by willpower232** [reply]
if you do `docker ps -a` do you see mariadb listed and running?

**Reply by webmagnets** [reply]
Yes, it is running.

![image](https://github.com/user-attachments/assets/db82faa0-b2fa-4272-8976-5dc51a958111)

**Reply by willpower232** [reply]
I think you need to `docker remove hopeful_brahmagupta` and then perhaps `postal logs worker` to see if you get any clues as to why it is restarting

**Reply by webmagnets** [reply]
Thank you. postal logs worker spewed out probably 100 or more screens of data.

Here is what happened after removing hopeful_brahmagupta and running postal initialize again...

![image](https://github.com/user-attachments/assets/49e96147-b449-4644-900a-a011a1fb2701)

**Reply by willpower232** [reply]
I guess a good test would be if the web UI also still works?

**Reply by webmagnets** [reply]
Going to email.nathancain.com in the web browser gives me this error...

```
This site can’t be reached
email.nathancain.com took too long to respond.
```

**Reply by willpower232** [reply]
well that sounds more like a firewall problem unless `postal logs web` says otherwise

**Reply by webmagnets** [reply]
![image](https://github.com/user-attachments/assets/8f6061e4-6858-4bc5-a856-0159c630f82d)

**Reply by willpower232** [reply]
Yeah so there must be a firewall or something blocking access to your servers port 443 then. Or your DNS is very incorrect

**Reply by webmagnets** [reply]
I ran `sudo ufw allow 443` but it didn't seem to help. Is there another way to unblock that port?

**Reply by willpower232** [reply]
You may also have a firewall with your server provider which needs unblocking.
